### ISSUE TITLE
Test that everything works when a schema is created or modified

### ISSUE THREAD

**Issue by psrpinto** [WordPress/try-wordpress#176]
Test cases:

- Add a field to an existing schema and make sure everything works
- Create a new schema under `schemas/subjects/` and make sure everything works

**Comment by psrpinto** [comment]
Created this ticket because it's an action we need to do, and it also serves as the "finish line" for the work related to having schemas drive the system.

**Comment by ashfame** [comment]
I did a round of testing. Created #178 and I believe schema is driving all aspects of frontend and backend correctly now.

Here is how I tested it:

I created a `test.json` under `schema/subjects` directory with the following content:

```
{
	"title": "Page-two",
	"slug": "page-two",
	"fields": {
		"title": {
			"description": "The title of the page2",
			"type": "text"
		},
		"content": {
			"description": "The body of the page2",
			"type": "html"
		}
	}
}
```

Ran extension: `npm run start:chrome`

Subject import option shows up:

<img width="324" alt="Image" src="https://github.com/user-attachments/assets/a1a570c2-7707-48ec-a063-9ae955a7b58f" />

Proceeding:

<img width="317" alt="Image" src="https://github.com/user-attachments/assets/0c98887a-c312-4c04-a080-f59facabab8d" />

Defining subject definition:

<img width="1192" alt="Image" src="https://github.com/user-attachments/assets/fe26a7cf-d44c-4355-a9a4-ee68ab8ad242" />

Preview:

<img width="1191" alt="Image" src="https://github.com/user-attachments/assets/44aa4e8d-9857-4792-915f-a934c0959b21" />

**Comment by psrpinto** [comment]
We identified an [issue](https://github.com/WordPress/try-wordpress/issues/182) with the import of blog posts, investigating.

**Comment by psrpinto** [comment]
Waiting for #188 to be merged to do a final test.

**Comment by psrpinto** [comment]
Here's a couple of screen recording that show the schemas in action. The first shows adding a field to a schema, the second shows adding a new schema. /cc @akirk 

Note that in the first recording the field is not shown on the preview and, in the second recording the preview 404s. This is expected at this point, this will be addressed in #139 .

## Adding a field
https://github.com/user-attachments/assets/d5d848b5-268a-47e8-b9af-f126c570b6ce

## Adding a subject type
https://github.com/user-attachments/assets/7a9cc903-1f75-4367-acb9-f17b44ee7730
